docs/VM-Extension.md

```diff
@@ -69,16 +85,42 @@ You can change to Azure Resource Manager mode by running:
 $ azure config mode arm
 ```
 
-You can deploy the OmsAgent Extension by running:
+You can deploy the OMS Agent for Linux Extension by running:
 ```
-$ azure vm extension set <resource-group> <vm-name> \
-OmsAgentForLinux Microsoft.EnterpriseCloud.Monitoring <version> \
---public-config-path public.json  \
---private-config-path protected.json
+$ azure vm extension set <resource-group> \
+                         <vm-name> \
```

Comment:
all vm extension examples already present in azure-vm extension docs. Can you update this doc to link them instead?